test/Templates.Test/Templates.Test.csproj

```diff
@@ -26,4 +25,11 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitRunnerVisualStudioPackageVersion)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can `Template.Test` reference `DotnetToolInstaller`? Running these tests require the repo to be built, but it would be nice to explicitly express the dependency